tools/rdm/TestDefinitions.py

```diff
@@ -603,7 +609,11 @@ def Test(self):
           advisory='Responder timed out to a command with PDL of %d' %
                    self.MAX_PDL),
     ])
-    self.SendRawGet(ROOT_DEVICE, self.pid, 'x' * self.MAX_PDL)
+    # Incrementing list, so we can find out which bit we have where in memory
+    data = ''
+    for i in xrange(0, self.MAX_PDL):
+      data += chr(i)
```

Comment:
This is going to break formatting (consider \n and null in the data).

Can we just use 0123456789 repeating  instead?